wwwroot/inc/snmp.php

```diff
@@ -2645,6 +2657,12 @@
 		'text' => 'WS-C2960G-8TC-L: 7 RJ-45/10-100-1000T(X) + 1 combo-gig',
 		'processors' => array ('catalyst-chassis-8-combo-1000SFP', 'catalyst-chassis-any-1000T'),
 	),
+	'9.1.917' => array
+	(
+    	'dict_key' => 1018,
+    	'text' => 'Cisco Catalyst WS-C4900M 20-port wirespeed 10/100/1000 (RJ-45), 4-port wirespeed 10GbE (X2), 8-port (2:1) 10GbE (X2)',
+    	'processors' => array ('catalyst-chassis-uplinks-10000X2','catalyst-chassis-any-1000T'),
```

Comment:
this is a modular switch, meaning it could have a variety of cards and ports installed.

https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-4900-series-switches/Prod_Bulletin_447737_Cat_4900M-Ex.html

the processors you defined may satisfy the requirements of the person who filed the ticket, but not cover the rest of the possibilities.  I'm hesitant to add partial support like this.  maybe @infrastation has an opinion on the topic

Reply:
"Cisco Systems, Inc. WS-C4900M 3 slot switch" is outside of the scope of this job, just skip all modular gear for now and resolve as many simple tickets as reasonably possible. Thanks.